rocAL/rocAL/source/loaders/video/video_loader.cpp

```diff
@@ -220,22 +220,22 @@ VideoLoader::load_routine()
             std::this_thread::sleep_for(std::chrono::seconds(1));
         }
     }
-    return VideoLoaderModuleStatus::OK;
+    return LoaderModuleStatus::OK;
```

Comment:
Loader module for Video had been removed ?
Same loader is used for both images & Videos ?

Reply:
Yes